cloud-etl/statements.sql

```diff
@@ -1,3 +1,3 @@
 CREATE STREAM eventlogs (eventSourceIP varchar, eventAction varchar, Result varchar, eventDuration bigint) WITH (kafka_topic='eventlogs', value_format='JSON');
 CREATE TABLE count_per_source WITH (KAFKA_TOPIC='COUNT_PER_SOURCE', PARTITIONS=6) AS SELECT eventSourceIP, count(*) as COUNT FROM eventlogs GROUP BY eventSourceIP EMIT CHANGES;
-CREATE TABLE sum_per_source WITH (KAFKA_TOPIC='SUM_PER_SOURCE', PARTITIONS=6, VALUE_FORMAT='AVRO') AS SELECT eventSourceIP, sum(eventDuration) as SUM FROM eventlogs WHERE Result='Pass' GROUP BY eventSourceIP EMIT CHANGES;
+CREATE TABLE sum_per_source WITH (KAFKA_TOPIC='SUM_PER_SOURCE', PARTITIONS=6, VALUE_FORMAT='AVRO') AS SELECT EVENTLOGS.EVENTSOURCEIP ROWKEY, AS_VALUE(EVENTLOGS.EVENTSOURCEIP) EVENTSOURCEIP, SUM(EVENTLOGS.EVENTDURATION) SUM FROM EVENTLOGS EVENTLOGS WHERE (EVENTLOGS.RESULT = 'Pass') GROUP BY EVENTLOGS.EVENTSOURCEIP EMIT CHANGES;
```

Comment:
@rspurgeon why is it necessary to prepend `EVENTLOGS.`?  It's inconsistent with above, and makes it a bit harder to read.  Is `AS_VALUE` not sufficient?

Reply:
it's the strict version of the command copied from C3 (in the running streams view).  I'll modify it to be the more 'human friendly' version and push it along w/ the change to docs i'm preparing

Reply:
@ybyzek please see updated version.  Thank you